### DISCUSSION TITLE
where find  Cloud Drive config example

### DISCUSSION THREAD

**Discussion by hktalent** [seaweedfs/seaweedfs#3892] [General]
* 1- where find  Cloud Drive config example?
* 2- A number of large files are 20G, stored on 20 nodes, and the number of copies is 5. Is that sharding stored on 20 nodes, or redundant, fault-tolerant, and sharding stored on 20 nodes?

**Comment by chrislusf** [comment]
1. https://github.com/seaweedfs/seaweedfs/wiki/Cloud-Drive-Quick-Setup
2. The question is unclear.
